jekyll/_docs/chromedriver-moving-elements.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: classic-docs
-title: ChromeDriver raises an 'Element is not clickable' exception
-description: Element no clickable issue
+title: "ChromeDriver raises an 'Element is not clickable' exception"
+description: "Element isn't Clickable Issue with ChromeDriver"
```

Comment:
might consider putting the single quotes in the description too around "Element isn't Clickable" - also "Issue" shouldn't be capitalized